akka-distributed-cluster-docs/src/main/paradox/guide/1-event-sourced-shopping-cart.md

```diff
@@ -34,15 +34,26 @@ Java
 
```

Comment:
The State has `customerId` and `customerCategory`. First I thought that was for the RES filter, but I think that is `vipCustomer`?

Reply:
Leftover from something that was before, I missed out on dropping it on the Scala side. We sort of re-introduce category in the RES sample for filtering but I made that a boolean for simplicity. Fixed in 4dd9c03f46eb1c14fa9d554e5284382c021ef678